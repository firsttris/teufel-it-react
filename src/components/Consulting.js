import React, { Component } from 'react';
import App from './App';
import Meeting from './../assets/images/meeting.jpg';
import Server from './../assets/images/server.jpg';
import PrinciplesJson from './../assets/json/consulting/principles.json'
import PortfolioJson from './../assets/json/consulting/portfolio.json'

class Consulting extends Component {
    render() {
        return (
            <div>
                <App image1={Meeting} image2={Server} image3={Meeting} portfolioJson={PortfolioJson} principlesJson={PrinciplesJson}></App>
            </div>
        );
    }
}

export default Consulting;