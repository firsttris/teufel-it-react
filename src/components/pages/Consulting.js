import React, {Component} from 'react';
import Parallax from './Parallax';
import Meeting from './../../../public/images/meeting2.jpg';
import Server from './../../../public/images/server2.jpg';
import ConsultingJson from './../../../public/pages/consulting.json';
import {Helmet} from "react-helmet";

export default class Consulting extends Component {
    render() {
        return (
            <div>
                <Helmet>
                    <title>Teufel IT, IT-Strategieberatung, Gesch채ftsanwendungen, Gesch채ftsprozesse</title>
                    <meta charSet="utf-8" />
                    <meta name="description" content="Teufel IT, Softwareentwicklung, Integration, IT-Beratung, IT-Strategieberatung, Gesch채ftsanwendungen, Gesch채ftsprozesse" />
                    <link rel="canonical" href="http://teufel-it.de/it-strategieberatung" />
                </Helmet>
                <Parallax image1={Meeting} image2={Server} image3={Meeting} json={ConsultingJson}/>
            </div>
        );
    }
}