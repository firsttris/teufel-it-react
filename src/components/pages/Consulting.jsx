import React, { Component } from 'react';
import Parallax from './Parallax.jsx';
import Meeting from 'images/meeting2.jpg';
import Server from 'images/server2.jpg';
import ConsultingJson from 'json/pages/consulting.json';
import { Helmet } from 'react-helmet';

class Consulting extends Component {
  render() {
    return (
      <div>
        <Helmet>
          <title>Teufel IT, IT-Strategieberatung, Gesch채ftsanwendungen, Gesch채ftsprozesse</title>
          <meta charSet="utf-8" />
          <meta
            name="description"
            content="Teufel IT, Softwareentwicklung, Integration, IT-Beratung, IT-Strategieberatung, Gesch채ftsanwendungen, Gesch채ftsprozesse"
          />
          <link rel="canonical" href="http://teufel-it.de/it-strategieberatung" />
        </Helmet>
        <Parallax image1={Meeting} image2={Server} image3={Meeting} json={ConsultingJson} />
      </div>
    );
  }
}

export default Consulting;
