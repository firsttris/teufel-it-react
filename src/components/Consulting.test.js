import React from 'react';
import Consulting from './Consulting';
import renderer from 'react-test-renderer';

test('Display Component', () => {
  const component = renderer.create(
    <Consulting></Consulting>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});