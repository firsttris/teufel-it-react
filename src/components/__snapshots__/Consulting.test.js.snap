// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Component 1`] = `
<div>
  <div
    className="container-fluid"
    style={
      Object {
        "backgroundImage": "url(lines.png)",
        "textAlign": "center",
      }
    }
  >
    <div
      className="row"
    >
      <div
        className="col"
        style={
          Object {
            "padding": 0,
          }
        }
      >
        <div
          style={
            Object {
              "backgroundAttachment": "fixed",
              "backgroundImage": "url(meeting.jpg)",
              "backgroundPosition": "center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "cover",
              "height": "550px",
              "minHeight": "400px",
              "opacity": "0.65",
              "position": "relative",
            }
          }
        />
      </div>
    </div>
    <div
      className="row py-3"
    >
      <div
        style={
          Object {
            "position": "absolute",
            "top": "200px",
            "width": "100%",
          }
        }
      >
        <div
          className="p-3"
          style={
            Object {
              "backgroundColor": "#4B4B4B",
              "display": "inline-block",
            }
          }
        >
          <div
            style={
              Object {
                "color": "#f00",
                "fontSize": "35px",
                "letterSpacing": "8px",
              }
            }
          >
            [ teufel it ]
          </div>
          <div
            className="pt-3"
            style={
              Object {
                "color": "white",
                "fontSize": "15px",
                "letterSpacing": "5px",
              }
            }
          >
            Software ist unser Handwerk
          </div>
        </div>
      </div>
    </div>
    <div
      className="row py-3"
    >
      <div
        className="col"
      >
        <div
          className="text-uppercase"
          style={
            Object {
              "fontSize": "30px",
              "letterSpacing": "10px",
            }
          }
        >
          portfolio
        </div>
        <p>
          Wir sind zuverlässiger Partner bei
        </p>
      </div>
    </div>
    <div
      className="row py-3"
    >
      <div
        className="col-xs-12 col-md-6 col-lg-4"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "height": "250px",
            "perspective": "800px",
          }
        }
      >
        <div
          style={
            Object {
              "backfaceVisibility": "hidden",
              "transform": "rotateY(0deg)",
              "transformStyle": "preserve-3d",
              "transition": "transform 1s linear",
              "width": "100%",
            }
          }
        >
          <div
            style={
              Object {
                "backfaceVisibility": "hidden",
                "position": "absolute",
                "top": "0",
                "width": "100%",
              }
            }
          >
            <i
              aria-hidden="true"
              className="fa fa-5x fa-history"
              style={
                Object {
                  "color": "#f00",
                  "fontSize": "10em",
                }
              }
            />
            <div
              className="text-uppercase"
              style={
                Object {
                  "fontSize": "1.5em",
                }
              }
            >
              Analyse
            </div>
          </div>
          <div
            style={
              Object {
                "backfaceVisibility": "hidden",
                "position": "absolute",
                "top": "0",
                "transform": "rotateY(180deg)",
                "width": "100%",
              }
            }
          >
            <div
              className="p-2"
              style={
                Object {
                  "alignItems": "center",
                  "border": "1px solid #f00",
                  "borderRadius": "5px",
                  "display": "flex",
                  "height": "175px",
                  "margin": "auto",
                  "width": "50%",
                }
              }
            >
              Bestandsaufnahme und Analyse des IT-Umfelds. Bewertung der aktuellen Strategie. Anwendungen und Prozessen.
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-xs-12 col-md-6 col-lg-4"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "height": "250px",
            "perspective": "800px",
          }
        }
      >
        <div
          style={
            Object {
              "backfaceVisibility": "hidden",
              "transform": "rotateY(0deg)",
              "transformStyle": "preserve-3d",
              "transition": "transform 1s linear",
              "width": "100%",
            }
          }
        >
          <div
            style={
              Object {
                "backfaceVisibility": "hidden",
                "position": "absolute",
                "top": "0",
                "width": "100%",
              }
            }
          >
            <i
              aria-hidden="true"
              className="fa fa-5x fa-cogs"
              style={
                Object {
                  "color": "#f00",
                  "fontSize": "10em",
                }
              }
            />
            <div
              className="text-uppercase"
              style={
                Object {
                  "fontSize": "1.5em",
                }
              }
            >
              Optimierung
            </div>
          </div>
          <div
            style={
              Object {
                "backfaceVisibility": "hidden",
                "position": "absolute",
                "top": "0",
                "transform": "rotateY(180deg)",
                "width": "100%",
              }
            }
          >
            <div
              className="p-2"
              style={
                Object {
                  "alignItems": "center",
                  "border": "1px solid #f00",
                  "borderRadius": "5px",
                  "display": "flex",
                  "height": "175px",
                  "margin": "auto",
                  "width": "50%",
                }
              }
            >
              Erstellung von Konzepten für Neuausrichtung und Optimierung.
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-xs-12 col-md-6 col-lg-4"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "height": "250px",
            "perspective": "800px",
          }
        }
      >
        <div
          style={
            Object {
              "backfaceVisibility": "hidden",
              "transform": "rotateY(0deg)",
              "transformStyle": "preserve-3d",
              "transition": "transform 1s linear",
              "width": "100%",
            }
          }
        >
          <div
            style={
              Object {
                "backfaceVisibility": "hidden",
                "position": "absolute",
                "top": "0",
                "width": "100%",
              }
            }
          >
            <i
              aria-hidden="true"
              className="fa fa-5x fa-github"
              style={
                Object {
                  "color": "#f00",
                  "fontSize": "10em",
                }
              }
            />
            <div
              className="text-uppercase"
              style={
                Object {
                  "fontSize": "1.5em",
                }
              }
            >
              open-source
            </div>
          </div>
          <div
            style={
              Object {
                "backfaceVisibility": "hidden",
                "position": "absolute",
                "top": "0",
                "transform": "rotateY(180deg)",
                "width": "100%",
              }
            }
          >
            <div
              className="p-2"
              style={
                Object {
                  "alignItems": "center",
                  "border": "1px solid #f00",
                  "borderRadius": "5px",
                  "display": "flex",
                  "height": "175px",
                  "margin": "auto",
                  "width": "50%",
                }
              }
            >
              Durch Open-Source Software und Cloudlösungen bieten wir unkonventionelle und moderne Lösungen
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col"
        style={
          Object {
            "padding": 0,
          }
        }
      >
        <div
          style={
            Object {
              "backgroundAttachment": "fixed",
              "backgroundImage": "url(server.jpg)",
              "backgroundPosition": "center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "cover",
              "height": "350px",
              "minHeight": "400px",
              "opacity": "0.65",
              "position": "relative",
            }
          }
        />
      </div>
    </div>
    <div
      className="row py-3"
      style={
        Object {
          "backgroundImage": "url(paper.png)",
          "color": "white",
        }
      }
    >
      <div
        className="col"
      >
        <div
          className="text-uppercase"
          style={
            Object {
              "fontSize": "30px",
              "letterSpacing": "10px",
            }
          }
        >
          ziele
        </div>
        <p>
          Wir arbeiten auf Basis von bewährten Konzepten
        </p>
      </div>
    </div>
    <div
      className="row py-3"
      style={
        Object {
          "backgroundImage": "url(paper.png)",
          "color": "white",
        }
      }
    >
      <div
        className="py-2 col-xs-12 col-md-6 col-lg-6"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <div
          style={
            Object {
              "alignItems": "center",
              "display": "inline-flex",
            }
          }
        >
          <i
            aria-hidden="true"
            className="fa fa-2x fa-money"
            style={
              Object {
                "color": "#f00",
                "transition": "color 2s linear 1s",
              }
            }
          />
          <span
            className="text-uppercase"
            style={
              Object {
                "fontSize": "20px",
                "marginLeft": "10px",
              }
            }
          >
            Ersparnis
          </span>
        </div>
        <p>
          Weniger Wartungskosten
        </p>
      </div>
      <div
        className="py-2 col-xs-12 col-md-6 col-lg-6"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <div
          style={
            Object {
              "alignItems": "center",
              "display": "inline-flex",
            }
          }
        >
          <i
            aria-hidden="true"
            className="fa fa-2x fa-sitemap"
            style={
              Object {
                "color": "#f00",
                "transition": "color 2s linear 1s",
              }
            }
          />
          <span
            className="text-uppercase"
            style={
              Object {
                "fontSize": "20px",
                "marginLeft": "10px",
              }
            }
          >
            Standards
          </span>
        </div>
        <p>
          Einsatz von bewährten Konzepten
        </p>
      </div>
      <div
        className="py-2 col-xs-12 col-md-6 col-lg-6"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <div
          style={
            Object {
              "alignItems": "center",
              "display": "inline-flex",
            }
          }
        >
          <i
            aria-hidden="true"
            className="fa fa-2x fa-puzzle-piece"
            style={
              Object {
                "color": "#f00",
                "transition": "color 2s linear 1s",
              }
            }
          />
          <span
            className="text-uppercase"
            style={
              Object {
                "fontSize": "20px",
                "marginLeft": "10px",
              }
            }
          >
            Einfach & Fair
          </span>
        </div>
        <p>
          Reduzierung der Komplexität Ihrer IT-Systeme
        </p>
      </div>
      <div
        className="py-2 col-xs-12 col-md-6 col-lg-6"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <div
          style={
            Object {
              "alignItems": "center",
              "display": "inline-flex",
            }
          }
        >
          <i
            aria-hidden="true"
            className="fa fa-2x fa-rocket"
            style={
              Object {
                "color": "#f00",
                "transition": "color 2s linear 1s",
              }
            }
          />
          <span
            className="text-uppercase"
            style={
              Object {
                "fontSize": "20px",
                "marginLeft": "10px",
              }
            }
          >
            Business
          </span>
        </div>
        <p>
          Konzentration auf Ihr Kerngeschäft
        </p>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col"
        style={
          Object {
            "padding": 0,
          }
        }
      >
        <div
          style={
            Object {
              "backgroundAttachment": "fixed",
              "backgroundImage": "url(meeting.jpg)",
              "backgroundPosition": "center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "cover",
              "height": "350px",
              "minHeight": "400px",
              "opacity": "0.65",
              "position": "relative",
            }
          }
        />
      </div>
    </div>
    <div
      className="row pt-3"
    >
      <div
        className="col"
      >
        <div
          className="text-uppercase"
          style={
            Object {
              "fontSize": "30px",
              "letterSpacing": "10px",
            }
          }
        >
          Kontakt
        </div>
        <p>
          Kontaktieren Sie mich über
        </p>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col"
        style={
          Object {
            "margin": "0 auto",
          }
        }
      >
        <a
          aria-hidden="true"
          className="fa fa-2x fa-xing-square"
          href="https://www.xing.com/profile/Tristan_Teufel"
          rel="noopener noreferrer"
          style={
            Object {
              "color": "#f00",
              "margin": "10px",
              "textDecoration": "none",
            }
          }
          target="_blank"
        >
           
        </a>
        <a
          aria-hidden="true"
          className="fa fa-2x fa-linkedin-square"
          href="https://de.linkedin.com/pub/tristan-teufel/8a/30a/b5"
          rel="noopener noreferrer"
          style={
            Object {
              "color": "#f00",
              "margin": "10px",
              "textDecoration": "none",
            }
          }
          target="_blank"
        >
           
        </a>
        <a
          aria-hidden="true"
          className="fa fa-2x fa-github"
          href="https://github.com/firsttris"
          rel="noopener noreferrer"
          style={
            Object {
              "color": "#f00",
              "margin": "10px",
              "textDecoration": "none",
            }
          }
          target="_blank"
        >
           
        </a>
      </div>
    </div>
    <div
      className="row pb-3"
    >
      <div
        className="col"
        style={
          Object {
            "margin": "0 auto",
          }
        }
      >
        <a
          className="btn btn-secondary"
          href="tel:+49 176 45744166"
          rel="noopener noreferrer"
          style={
            Object {
              "margin": "10px",
            }
          }
          target="_blank"
        >
          +49 176 45744166
        </a>
        <a
          className="btn btn-secondary"
          href="mailto:info@teufel-it.de"
          rel="noopener noreferrer"
          style={
            Object {
              "margin": "10px",
            }
          }
          target="_blank"
        >
          info@teufel-it.de
        </a>
      </div>
    </div>
    <div
      className="row py-3"
      style={
        Object {
          "backgroundImage": "url(paper.png)",
          "color": "white",
        }
      }
    >
      <div
        className="col-xs-12 col-md-4 col-lg-4"
      >
        <div>
          Inhaber:
        </div>
        <div>
          Tristan Teufel
        </div>
        <div>
          Werkstr.10a
        </div>
        <div>
          D-77815 Bühl
        </div>
      </div>
      <div
        className="col-xs-12 col-md-4 col-lg-4"
      >
        <span>
          Build with 
          <a
            href="https://facebook.github.io/react/"
            rel="noopener noreferrer"
            style={
              Object {
                "color": "#61DAFB",
                "textDecoration": "none",
              }
            }
            target="_blank"
          >
            <b>
              React
            </b>
          </a>
        </span>
        <img
          alt="logo"
          className="App-logo"
          src="logo.svg"
        />
        <div>
          <div>
            Quellen für Bilder und Icons:
          </div>
          <a
            href="http://unsplash.com"
            rel="noopener noreferrer"
            style={
              Object {
                "color": "#61DAFB",
                "textDecoration": "none",
              }
            }
            target="_blank"
          >
            unsplash.com 
          </a>
          <a
            href="http://shutterstock.com"
            rel="noopener noreferrer"
            style={
              Object {
                "color": "#61DAFB",
                "textDecoration": "none",
              }
            }
            target="_blank"
          >
            shutterstock.com 
          </a>
          <a
            href="http://fontawesome.io"
            rel="noopener noreferrer"
            style={
              Object {
                "color": "#61DAFB",
                "textDecoration": "none",
              }
            }
            target="_blank"
          >
            fontawesome.io
          </a>
        </div>
      </div>
      <div
        className="col-xs-12 col-md-4 col-lg-4"
      >
        <div>
          Kontakt:
        </div>
        <div>
          <a
            href="tel:+4917645744166"
            style={
              Object {
                "color": "#61DAFB",
                "textDecoration": "none",
              }
            }
          >
            +49 (0) 176 457 44 1 66
          </a>
        </div>
        <div>
          <a
            href="mailto:info@teufel-it.de"
            style={
              Object {
                "color": "#61DAFB",
                "textDecoration": "none",
              }
            }
          >
            info@teufel-it.de
          </a>
        </div>
        <div>
          Finanzamt Baden-Baden 36216/14616
        </div>
      </div>
    </div>
  </div>
</div>
`;
